docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -0,0 +1,190 @@
+* If Azure CLI 2.66.0 is the last release of Azure CLI before breaking change in the fourth quarter of the calendar year, it's supported until the release of Azure CLI 2.79.0.
+
+## Breaking changes
+
+Major version releases of Azure CLI introduce breaking changes and occur up to two times per year. Breaking changes can occur at any point for preview extensions. Non-GA extensions aren't required to adhere to breaking change policies.
+
+For detailed information about breaking changes in Azure CLI, see [Release notes](./release-notes-azure-cli.md).
+
+## Supported environments
+
+The Azure CLI runs on multiple operating systems (OS) and architectures. For a complete list, see [Install the Azure CLI](./install-azure-cli.md). To be supported by Microsoft, the following criteria must be met:
+
+* The version of Azure CLI is supported.
+* The version of the OS is currently in mainstream support by the OS publisher.
+* The dependencies required by the current version of Azure CLI are supported on the OS.
+
+The Azure CLI ends support for a platform when one of the following conditions is met:
+
+* The OS reaches its end of life as defined by the platform owner.
+* The version of Python required by Azure CLI reaches its end of life, is no longer supported on the OS, or has an unfixed critical security issue.
+
+Azure CLI announces the end of support of an OS or one of its dependencies within three months of the public announcement of the retirement.
+
+## Python dependency
+
+Azure CLI depends on Python version 3.8 or above. The following table summarizes the expected end of support for each version of Python:
+
+|Python version|End of support date|
+|-|-|
+|3.8|October 2024 |
+|3.9|October 2025 |
+|3.10|October 2026 |
+|3.11|October 2027 |
+
+Azure CLI can only be supported on operating systems where the above versions of Python are supported.
+
+* **Windows:** The currently supported versions of Windows client and server meet Python version requirements.
+* **macOS:** The currently supported version of macOS 10.9 and above meet Python version requirements.
+* **Linux:**
+  * Each supported operating system has a lifecycle defined by its sponsor organization.
+  * Support is typically removed when an operating system goes out of mainline support, at which time we stop testing and supporting it.
+  * Here are the supported Linux operating systems for the Azure CLI:
+
+    |Operating system|Version|End of support|
+    |-|-|-|
+    | Ubuntu| 20.04 LTS| April 2025 |
+    | | 22.04 LTS | April 2027 |
+    | Debian | 10 | June 2024 |
+    | | 11 | |
+    | | 12 | |
+    | RHEL | 8 | May 31, 2029 |
+    | | 9 | May 31, 2032 |
+    | CentOS Stream | 9 | Estimated 2027 |
+    | Mariner | 2.0 | |
+    | Alpine | 3.17 | November 22, 2024 |
+    | | 3.16 | May 23, 2024 |
+
+    > [!NOTE]
+    > On RHEL 7, only Azure CLI 2.38 is supported and will receive security fixes until June 30, 2024.
```

Comment:
It is EOL now.

We also need to modify this statement in doc:
> Azure CLI [2.38.2](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#june-21-2024) (released on June 21, 2024) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft isn't providing other updates or bug fixes for these distributions.

Reply:
@bebound, I've made this change in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455). Please leave additional comments in the new PR.